app/src/cli/mod.rs

```diff
@@ -16,11 +16,13 @@ pub use project_path::*;
 use self::assets::AssetCommand;
 
 #[derive(Parser, Clone)]
-#[command(author, version, about, long_about = None)]
```

Comment:
You can specify an expression to use for `version` here instead of adding `--version` as a bool: https://docs.rs/clap/latest/clap/_derive/index.html#command-attributes

Reply:
thanks! what a detour I made 😅